python/torch_mlir/dialects/torch/importer/jit_ir/csrc/torch_to_mlir_utils.cpp

```diff
@@ -408,14 +408,29 @@ MlirAttribute torch_mlir::importAttribute(MlirLocation loc,
 MlirLocation torch_mlir::getMlirLocationFromNode(MlirContext context,
                                                  torch::jit::Node *node) {
   auto flc = node->sourceRange().file_line_col();
+  const c10::FunctionSchema *schema = node->maybeSchema();
+  if (!flc && !schema)
+    return mlirLocationUnknownGet(context);
+  MlirLocation loc;
```

Comment:
nit: I think this case is handled by the logic below. Can replace with `MlirLocation loc = mlirLocationUnknownGet(context);`